packages/venia-ui/lib/components/CartPage/GiftCards/__tests__/__snapshots__/giftCard.spec.js.snap

```diff
@@ -16,9 +22,15 @@ Array [
 
 exports[`it renders correctly 1`] = `
 Array [
-  <span>
-    unit test card code
-  </span>,
+  <div>
+    <span>
+      unit test card code
+    </span>
+    <span>
+      Balance: 
```

Comment:
There is no balance in these snaps, not even a zero. Expected?

Reply:
Expected because the value is rendered via the `Price` component, which gets mocked here to an empty `<div />` (see next line).